sites-available/ssl.no-default

```diff
@@ -6,7 +6,9 @@
 # returning 444 "no response".
 
 server {
+  listen [::]:443 ssl default_server;
```

Comment:
I feel like `deferred` should be added here or excluded from the above `listen 80`